concourse/steps/release.py

```diff
@@ -1043,7 +1070,7 @@ def release_and_prepare_next_dev_cycle(
                 slack_cfg_name=slack_cfg_name,
                 slack_channel=slack_channel,
                 release_version=release_version,
-                release_notes=release_notes,
+                release_notes=rls_notes,
```

Comment:
>one might consider to use ...model.EnumName.DEFAULT(.value) to have an additional typehint, which may be helpful for future refactorings (consider grepping for all occurrences..); that being said, it might be preferable to expect the enum-value, and not a str, rather than converting within this function

Personally I'd agree with typing this more strongly. But that would break the existing convention here.
While I am intentionally refraining from using our transaction-coding, I'm still trying to adhere to the existing style for the other parts. At least for this PR.

> naming: I would prefer to not abreviate release w/ rls
That was clashing with the import of the release-notes coding. I'll think about it some more.